RZAssemblage/RZArrayAssemblage.m

```diff
@@ -0,0 +1,170 @@
+//
+//  RZArrayAssemblage.m
+//  RZAssemblage
+//
+//  Created by Brian King on 3/21/15.
+//  Copyright (c) 2015 Raizlabs. All rights reserved.
+//
+
+#import "RZArrayAssemblage.h"
+#import "RZAssemblage+Private.h"
+
+NSString *const RZAssemblageUpdateKey = @"RZAssemblageUpdateKey";
+static char RZAssemblageUpdateContext;
```

Comment:
More convenient to use `static void* const RZAssemblageUpdateContext = (void *)&RZAssemblageUpdateContext `, then don't need to add the reference later.